### PR TITLE
Add `ignored` option.

### PR DESCRIPTION
**Pull request by wtgtybhertgeghgtwtg**
Retrying #77 for CI.

### CODE REVIEW

**Comment by amasad** [comment]
strange, there is still no ci. Maybe you haven't rebased?

**Comment by wtgtybhertgeghgtwtg** [comment]
[Apparently CircleCI only runs after the PR's been merged.](https://discuss.circleci.com/t/pull-requests-not-triggering-build/1213/4)

**Comment by amasad** [comment]
oh look, it's somehow now running

**Comment by amasad** [comment]
screen shot 2016-12-20 at 7 44 05 pm

**Comment by wtgtybhertgeghgtwtg** [comment]
Added fix for `watchman_walker`.  CircleCI doesn't seem to want to trigger when I merge a PR.

**Comment by wtgtybhertgeghgtwtg** [comment]
Would you trigger the CI for https://github.com/amasad/sane/pull/79/commits/e6dfd278325a193497daf9b3ef9318ac2264c53a?

File: src/common.js [hunk 1]
```diff
@@ -40,22 +46,21 @@ exports.assignOptions = function(watcher, opts) {
  * @public
  */
 
-exports.isFileIncluded = function(globs, dot, relativePath) {
+exports.isFileIncluded = function(globs, dot, doIgnore, relativePath) {
   var matched;
   if (globs.length) {
     for (var i = 0; i < globs.length; i++) {
-      if (minimatch(relativePath, globs[i], {dot: dot})) {
+      console.log(relativePath, !doIgnore(relativePath));
```
Comment (amasad) [inline src/common.js]: can you please remove the console.log

**Comment by amasad** [comment]
I don't know why it's acting up on this PR. I'll just merge it and have it test on master after you fix my comment.

**Comment by wtgtybhertgeghgtwtg** [comment]
`console.log` removed.  I can't believe I forgot to take that out.

**Comment by wtgtybhertgeghgtwtg** [comment]
If this is passing the CI, can it be published soon?  It's needed for https://github.com/webpack/watchpack/pull/41.

**Comment by amasad** [comment]
@wtgtybhertgeghgtwtg published: v1.5
Can you document the feature in Readme? You already have commit access.

**Pull request merged**